### PR TITLE
Fix golint issues for pkg/volume/rbd

### PR DESCRIPTION
**Pull request by pohsien324**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes `pkg/volume/rbd` golint issues.

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by jsafrane** [comment]
Can you please remove `pkg/volume/rbd` from `hack/.golint_failures`, so it is checked in verify tests and we keep the file lint-clean?

**Comment by pohsienshih** [comment]
@jsafrane  Thanks, I have deleted it.

**Comment by jsafrane** [comment]
You can see there is much more lint issues in RBD code, camelCase is just a small portion. Would you mind fixing all of them? See pull-kubernetes-verify output.

**Comment by pohsienshih** [comment]
@jsafrane Oh, I see. I'll take it, thanks!

**Comment by pohsienshih** [comment]
@jsafrane Updated. PTAL.

File: pkg/volume/rbd/rbd_test.go [hunk 1]
```diff
@@ -131,7 +131,7 @@ func TestCanSupport(t *testing.T) {
 	}
 }
 
-type fakeDiskManager struct {
+type FakeDiskManager struct {
```
Comment (jsafrane) [inline pkg/volume/rbd/rbd_test.go]: Why this type needs to be public?

File: pkg/volume/rbd/rbd_util.go [hunk 2]
```diff
@@ -271,16 +270,18 @@ func makeVDPDNameInternal(host volume.VolumeHost, pool string, image string) str
 	return filepath.Join(host.GetVolumeDevicePluginDir(rbdPluginName), pool+"-image-"+image)
 }
 
-// RBDUtil implements diskManager interface.
-type RBDUtil struct{}
+// Util implements diskManager interface.
+type Util struct{}
```
Comment (jsafrane) [inline pkg/volume/rbd/rbd_util.go]: Does `Util` need to be public? It seems to be used only in this package.

**Review by jsafrane — COMMENTED** [review]
Good job! I have only couple of questions.

File: pkg/volume/rbd/rbd_util.go (hunk 2 above)
Comment (pohsienshih) [reply pkg/volume/rbd/rbd_util.go]: Oops, I only noticed the function name but didn't notice that it is only used in this package. It seems like this function doesn't need to be public, I will modify it, thank you.

File: pkg/volume/rbd/rbd_test.go (hunk 1 above)
Comment (pohsienshih) [reply pkg/volume/rbd/rbd_test.go]: Sorry for the late reply. There is a golint problem with exported func `NewFakeDiskManager` that indicates it returning unexported type `*rbd.fakeDiskManager`. So I fixed this problem with exported it.

But after receiving your reminder, I found that the func `NewFakeDiskManager `is only used in this package. It is better to revoke the exportation of  func `NewFakeDiskManager` than export type `fakeDiskManager`?

Comment (jsafrane) [reply pkg/volume/rbd/rbd_test.go]: `newFakeDiskManager` /`fakeDiskManager` sounds better, IMO.

Comment (pohsienshih) [reply pkg/volume/rbd/rbd_test.go]: Okay, I got it!  I will update it, thanks.

File: pkg/volume/rbd/rbd_util.go (hunk 2 above)
Comment (pohsienshih) [reply pkg/volume/rbd/rbd_util.go]: Updated. PTAL. Thanks.

File: pkg/volume/rbd/rbd_test.go (hunk 1 above)
Comment (pohsienshih) [reply pkg/volume/rbd/rbd_test.go]: Updated. PTAL. Thanks.

Comment (pohsienshih) [reply pkg/volume/rbd/rbd_test.go]: Updated. Thanks.

**Comment by pohsienshih** [comment]
ping @jsafrane. 
I have updated the PR.  PTAL. Thanks!

**Comment by jsafrane** [comment]
/lgtm
/approve
Thanks for your PR!

**Comment by jsafrane** [comment]
/assign @cblecker @BenTheElder 
for one line `.golint_failures` approval

**Comment by cblecker** [comment]
/approve
for hack/

**Pull request merged**